Util/FileHandler.py

```diff
@@ -0,0 +1,39 @@
+import os
+import uuid
+
+from PIL import Image
+from PIL.ExifTags import GPSTAGS, TAGS
+
+
+def save(image: Image, filename: str) -> str:
+    # TODO replace with S3
+    if not os.path.exists('Data/'):
+        os.makedirs('Data')
+
+    _, ext = filename.split(".")
+    filename = uuid.uuid4().hex
```

Comment:
I see the utility in using a UUID, but we could avoid storing identical images by using a hash. What's the number of images we're typically ingesting here?

Reply:
That's a good idea, I was mostly concerned with name collisions rather than identical content. I'm not sure how many images we expect to ingest, I don't think we've considered that yet.